src/Console/Command/AddPrefixCommand.php

```diff
@@ -59,22 +61,29 @@ protected function configure()
             ->setName('add-prefix')
             ->setDescription('Goes through all the PHP files found in the given paths to apply the given prefix to namespaces & FQNs.')
             ->addArgument(
-                self::PREFIX_ARG,
-                InputArgument::REQUIRED,
-                'The namespace prefix to add'
-            )
-            ->addArgument(
                 self::PATH_ARG,
-                InputArgument::REQUIRED | InputArgument::IS_ARRAY,
+                InputArgument::IS_ARRAY,
                 'The path(s) to process.'
             )
             ->addOption(
-                self::OUTPUT_DIR,
+                self::PREFIX_OPT,
+                'p',
+                InputOption::VALUE_REQUIRED,
+                'The namespace prefix to add'
+            )
+            ->addOption(
+                self::OUTPUT_DIR_OPT,
                 'o',
                 InputOption::VALUE_REQUIRED,
                 'The output directory in which the prefixed code will be dumped.',
                 'build'
             )
+            ->addOption(
+                self::FORCE_OPT,
+                'f',
+                InputOption::VALUE_NONE,
+                'Deletes any existing content in the output directory without any warning'
```

Comment:
Err not really. I think integration tests would be better here given how FS can be troublesome, but there's some cases where mocking the FS would be more appropriate (e.g. when you change the perm to test the `is_writeable()`. I kinda wasted quite some time with that FS thing which is why I dropped it and checked manually for now